lib/ansible/modules/storage/netapp/na_ontap_ldap.py

```diff
@@ -0,0 +1,226 @@
+#!/usr/bin/python
+'''
+(c) 2018-2019, NetApp, Inc
+GNU General Public License v3.0+
+(see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+'''
+
+from __future__ import absolute_import, division, print_function
+import traceback
+import ansible.module_utils.netapp as netapp_utils
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils._text import to_native
+from ansible.module_utils.netapp_module import NetAppModule
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'certified'}
+
+DOCUMENTATION = '''
+
+module: na_ontap_ldap
+
+short_description: NetApp ONTAP LDAP
+extends_documentation_fragment:
+    - netapp.na_ontap
+version_added: '2.9'
+author: Milan Zink (@zeten30) <zeten30@gmail.com>/<mzink@redhat.com>
+
+description:
+- Create, modify or delete LDAP on NetApp ONTAP SV/vserver
+
+options:
+
+  state:
+    description:
+    - Whether the LDAP is present or not.
+    choices: ['present', 'absent']
+    default: 'present'
+    type: str
+
+  vserver:
+    description:
+    - vserver/svm configured to use LDAP
+    required: true
+    type: str
+
+  client_config:
```

Comment:
I would prefer to say “name”.  So each resource has a name, independently of the ZAPI internals.

Reply:
@lonico Ok, no issues with changing 'client_config' -> 'name'. I'll do it.

Reply:
I've just pushed updated code. Please check it.